Functions/TestResults.ps1

```diff
@@ -588,7 +588,7 @@ function Write-NUnitTestCaseAttributes($TestResult, [System.Xml.XmlWriter] $XmlW
             $XmlWriter.WriteAttributeString('result', 'Failure')
             $XmlWriter.WriteAttributeString('executed', 'True')
             $XmlWriter.WriteStartElement('failure')
-            $xmlWriter.WriteElementString('message', $TestResult.FailureMessage)
+            $xmlWriter.WriteElementString('message', $TestResult.FailureMessage.Replace("`e",'&27;'))
```

Comment:
`` `e`` escape sequence is a recent addition to PowerShell, prior version 6 it produces `e` (as escaping any other non-special letter) which will replace incorrectly. Please use a different way of providing the grave accent, for example a non-expandable string `` '`' ``.

Reply:
I just realized you meant `Escape` as in escape key, not escape sequence character (grave accent `` ` ``),  fixing.